### ISSUE TITLE
Multi Line Metrics Window

### ISSUE THREAD

**Issue by pathumego** [fontforge/fontforge#2324]
Self explanatory, ability to have multiple lines of glyphs in Metrics window. Im still thinking about details.

- Hitting Enter/Return would jump to next line?
- How will the text input box work with this?
- Or just use Wordlists?

**Comment by davelab6** [comment]
> Hitting Enter/Return would jump to next line?
> How will the text input box work with this?

Input text normally has `\n` to make a linebreak, and hitting enter would insert that string I guess. 

And otherwise it would wrap to the window border in a simple way

**Comment by larsenwork** [comment]
@pathumego great with new and better functions in general but I think it's important that step 1 in the menu makeover is to clean up and reorganise the existing and then wait with issues like this to "menu restructure - 2"
Not that we can't think further but I think it's best to not to try and do too much at once.

**Comment by pathumego** [comment]
@larsenwork Agree.. I will group some of the other UI improvements into a separate milestone, that we can work towards later.

**Comment by larsenwork** [comment]
@pathumego great, I said it because I saw you added this to milestone 1 :smile:

**Comment by pathumego** [comment]
yeah i figured :) Anyway after the menu restructuring @lpsandaruwan might have some time / continue contributing. So if we have a list of UI/UX improvements lined up in issues, organised in priority into milestones, then he can hammer on them. We will keep these milestones separate from regular releases and merge accordingly (So regular releases don't get dragged on..)

@davelab6 and I discussed a few months ago about the areas that we could improve, thats the wishlist doc. Metrics and Layers Pallet seems that they can use some UX love. Maybe that is something we could work on after the whole facelift is over :)

**Comment by larsenwork** [comment]
:+1:

**Comment by brigazvi** [comment]
How complicated should it be to add this feature?
